bindings/cs/MongoDB.Crypt.Test32/MongoDB.Crypt.Test32.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net452;netcoreapp2.1</TargetFrameworks>
+        <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1;netcoreapp2.2</TargetFrameworks>
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="../MongoDB.Crypt/MongoDB.Crypt.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="FluentAssertions" Version="5.6.0" />
+        <PackageReference Include="MongoDB.Bson" Version="2.8.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
+        <PackageReference Include="xunit" Version="2.4.0" />
+        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    </ItemGroup>
+
+
+    <ItemGroup Condition="'$(TargetFramework)'=='net452'">
+        <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    </ItemGroup>
+
+    <PropertyGroup>
+        <CMakeCurrentSourceDir>.</CMakeCurrentSourceDir>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+      <PlatformTarget>x86</PlatformTarget>
```

Comment:
This forces the test project to be 32 bit

Reply:
I don't see any tests in this project.

Reply:
There's a file called `BasicTests`.cs that should have one test

Reply:
Missed that. Sorry.

Reply:
No worries! 😺